internal/endian_le.go

```diff
@@ -0,0 +1,10 @@
+//go:build amd64 || arm64
+// +build amd64 arm64
+
+package internal
+
+import "encoding/binary"
+
+var NativeEndian binary.ByteOrder = binary.LittleEndian
```

Comment:
Please duplicate the comments so that generated documentation is consistent.

Reply:
Done!